bci_tester/data.py

```diff
@@ -664,6 +664,21 @@ def create_BCI(
     for mariadb_client_ver, os_versions in _MARIADB_VERSION_OS_MATRIX
 ]
 
+POSTFIX_CONTAINERS = [
+    create_BCI(
+        build_tag=f"{APP_CONTAINER_PREFIX}/postfix:{postfix_ver}",
+        bci_type=ImageType.APPLICATION,
+        available_versions=os_versions,
+        forwarded_ports=[PortForwarding(container_port=25)],
+        extra_environment_variables={"SERVER_HOSTNAME": "localhost"},
+    )
+    for postfix_ver, os_versions in (
+        (3.7, ["15.5"]),
```

Comment:
please remove the 15.5 container. as we haven't really released it for 15.6 yet either, we don't need to add new containers to 15.5 anymore.

Reply:
removed 15.5 container